Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -910,13 +910,15 @@ static bool refersToAllAspects(WGPUTextureFormat format, WGPUTextureAspect aspec
     auto aspectSpecificFormat = Texture::aspectSpecificFormat(destinationTexture.format(), destination.aspect);
     auto blockSize = Texture::texelBlockSize(aspectSpecificFormat);
     switch (destinationTexture.dimension()) {
-    case WGPUTextureDimension_1D:
-        sourceBytesPerRow = std::min<uint32_t>(sourceBytesPerRow, blockSize * m_device->limits().maxTextureDimension1D);
-        break;
+    case WGPUTextureDimension_1D: {
+        auto checkedBlockSizeTimesTextureDimension = checkedProduct<uint32_t>(blockSize, m_device->limits().maxTextureDimension1D);
```

Comment:
these probably technically aren't needed, since neither value originates from or is impacted by values passed over IPC